src/components/pages/quickstart/Prerequisites/openai.mdx

```diff
@@ -32,11 +32,16 @@ Below we show you Python and JavaScript installation commands, but you can find
 
 ### Set up your OpenAI client
 
-To connect with Arcade's enhanced capabilities, configure your client with two essential pieces:
+To use the OpenAI client with Arcade's enhanced capabilities, configure your client with two essential values:
 
-1. Point your client to Arcade's endpoint
+1. Point your client to Arcade's endpoint (base URL)
 2. Use your Arcade API key
 
+First, set the `ARCADE_API_KEY` environment variable:
+```bash
+export ARCADE_API_KEY=your_api_key_here
+```
```

Comment:
In this case it feels necessary to tell the user to `export` this because all of the code snippets below do `process.env` or `os.environ`